### PR TITLE
Update Set-ADReplicationSite.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected Kerberos consistency checker  to Knowledge Consistency Checker

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 1]
```diff
@@ -602,7 +602,7 @@ Accept wildcard characters: False
 ```
 
 ### -TopologyCleanupEnabled
-When enabled,  this optional parameter prevents the Kerberos consistency checker (KCC) from removing connection objects that it does not need.
+When enabled,  this optional parameter prevents the Knowledge consistency checker (KCC) from removing connection objects that it does not need.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:605]:
```suggestion
When enabled,  this optional parameter prevents the Knowledge Consistency Checker (KCC) from removing connection objects that it does not need.
```

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 2]
```diff
@@ -619,7 +619,7 @@ Accept wildcard characters: False
 ```
 
 ### -TopologyDetectStaleEnabled
-This parameter option prevents the Kerberos consistency checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
+This parameter option prevents the Knowledge consistency checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:622]:
```suggestion
This parameter option prevents the Knowledge Consistency Checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
```

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 3]
```diff
@@ -635,7 +635,7 @@ Accept wildcard characters: False
 ```
 
 ### -TopologyMinimumHopsEnabled
-When enabled, this parameter prevents the Kerberos consistency checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
+When enabled, this parameter prevents the Knowledge consistency checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:638]:
```suggestion
When enabled, this parameter prevents the Knowledge Consistency Checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
```

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 4]
```diff
@@ -729,7 +729,7 @@ Accept wildcard characters: False
 ```
 
 ### -WindowsServer2003KCCBehaviorEnabled
-Implements Kerberos consistency checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
+Implements Knowledge consistency checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:732]:
```suggestion
Implements Knowledge Consistency Checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
```

File: docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md [hunk 5]
```diff
@@ -760,7 +760,7 @@ Accept wildcard characters: False
 ```
 
 ### -WindowsServer2003KCCSiteLinkBridgingEnabled
-When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Kerberos consistency checker (KCC) control of the ability to enable or disable site link bridging.
+When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Knowledge consistency checker (KCC) control of the ability to enable or disable site link bridging.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Set-ADReplicationSite.md:763]:
```suggestion
When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Knowledge Consistency Checker (KCC) control of the ability to enable or disable site link bridging.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md [hunk 6]
```diff
@@ -133,7 +133,7 @@ Accept wildcard characters: False
 ```
 
 ### -AutomaticInterSiteTopologyGenerationEnabled
-Indicates whether the cmdlet prevents the Kerberos consistency checker (KCC) that functions as the intersite topology generator (ISTG) from generating connections for intersite replication.
+Indicates whether the cmdlet prevents the Knowledge consistency checker (KCC) that functions as the intersite topology generator (ISTG) from generating connections for intersite replication.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md:136]:
```suggestion
Indicates whether the cmdlet prevents the Knowledge Consistency Checker (KCC) that functions as the intersite topology generator (ISTG) from generating connections for intersite replication.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md [hunk 7]
```diff
@@ -513,7 +513,7 @@ Accept wildcard characters: False
 
 ### -TopologyCleanupEnabled
 Indicates whether the cmdlet enables topology cleanup.
-When enabled,  this optional parameter prevents the Kerberos consistency checker (KCC) from removing connection objects that it does not need.
+When enabled,  this optional parameter prevents the Knowledge consistency checker (KCC) from removing connection objects that it does not need.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md:516]:
```suggestion
When enabled,  this optional parameter prevents the Knowledge Consistency Checker (KCC) from removing connection objects that it does not need.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md [hunk 8]
```diff
@@ -531,7 +531,7 @@ Accept wildcard characters: False
 
 ### -TopologyDetectStaleEnabled
 Indicates whether the cmdlet enables topology detect stale.
-This parameter option prevents the Kerberos consistency checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
+This parameter option prevents the Knowledge consistency checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md:534]:
```suggestion
This parameter option prevents the Knowledge Consistency Checker (KCC) from excluding servers that are unreachable from the topology; that is, the KCC does use an alternate server to reroute replication.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md [hunk 9]
```diff
@@ -548,7 +548,7 @@ Accept wildcard characters: False
 
 ### -TopologyMinimumHopsEnabled
 Indicates whether the cmdlet enables topology minimum hops.
-When enabled, this parameter prevents the Kerberos consistency checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
+When enabled, this parameter prevents the Knowledge consistency checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md:551]:
```suggestion
When enabled, this parameter prevents the Knowledge Consistency Checker (KCC) from generating optimizing connections in the ring topology of intrasite replication.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md [hunk 10]
```diff
@@ -644,7 +644,7 @@ Accept wildcard characters: False
 ```
 
 ### -WindowsServer2003KCCBehaviorEnabled
-Indicates whether the cmdlet implements Kerberos consistency checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
+Indicates whether the cmdlet implements Knowledge consistency checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md:647]:
```suggestion
Indicates whether the cmdlet implements Knowledge Consistency Checker (KCC) operation that is consistent with Windows Server 2003 forest functional level.
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md [hunk 11]
```diff
@@ -677,7 +677,7 @@ Accept wildcard characters: False
 
 ### -WindowsServer2003KCCSiteLinkBridgingEnabled
 Indicates whether the cmdlet enables site link bridging.
-When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Kerberos consistency checker (KCC) control of the ability to enable or disable site link bridging.
+When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Knowledge consistency checker (KCC) control of the ability to enable or disable site link bridging.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Set-ADReplicationSite.md:680]:
```suggestion
When the forest functional level Windows Server 2003 or Windows Server 2003 interim is in effect, provides Knowledge Consistency Checker (KCC) control of the ability to enable or disable site link bridging.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**